examples/config-gc-periodic.json

```diff
@@ -4,13 +4,13 @@
         "rootDirectory": "/tmp/zot",
         "gc": true,
         "gcDelay": "1h",
-        "gcInterval": "24h",
+        "gcInterval": "1h",
```

Comment:
I wouldn't call this gc. Garbage is something that is deleted and not used and we are doing cleanup. That cleanup should happen on a continuous basis, not on a time period. For a given repository, we should define retention rules. This is explicitly at a repository level or at an image level.

Reply:
> That cleanup should happen on a continuous basis, not on a time period.

That doesn't really work for this.  We would have to, at startup, set a timer for every image and possibly reset the timer on every access, then delete the image when the timer goes off.  That could be hundreds of thousands of timers.  Doing a scan every 24h makes a lot more sense here.